cyfhir/src/test/java/com/Optum/CyFHIR/functions/BundleTest.java

```diff
@@ -29,10 +30,10 @@
 class BundleTest {
 
     @Container
-    private static final GenericContainer neo4j = new GenericContainer<>("neo4j:4.1.0")
-            .withEnv("NEO4J_AUTH", "neo4j/password")
+    private static final GenericContainer<?> neo4j = new GenericContainer<>(DockerImageName.parse("neo4j:4.2.7"))
+            .withExposedPorts(7687).withEnv("NEO4J_AUTH", "neo4j/password")
```

Comment:
bump withEnv to next line for consistency

Reply:
I actually had it bumped buy the mvn tdy package keeps moving it up